src/test/java/training/article/update.feature

```diff
@@ -0,0 +1,39 @@
+Feature: update an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+
+    * def token = "Bearer " + login.response.user.token
+    * def create = call read('create.feature@flow')
```

Comment:
do a GET of all the articles and after that get a random slug of this response to update.

Reply:
1. GET of all articles. (it returns an array)
2. choose a random article.
3. Do the update (PUT).
4. GET that article.
5. Match responses between steps 3 and 4.